modules/flowable-http/src/test/java/org/flowable/http/cmmn/CmmnHttpTaskTest.java

```diff
@@ -79,11 +79,11 @@ public void testGetWithResponseHandler() {
         CaseInstance caseInstance = createCaseInstance();
 
         Map<String, Object> variables = cmmnRule.getCmmnRuntimeService().getVariables(caseInstance.getId());
+        Map<String, String> names = new HashMap<>();
+        names.put("firstName", "John");
+        names.put("lastName", "Doe");
         assertThat(variables)
-                .containsExactly(
-                        entry("firstName", "John"),
-                        entry("lastName", "Doe")
-                );
+                .containsExactlyInAnyOrderEntriesOf(names);
```

Comment:
Thanks for looking into this @dbmalkovsky. 

Just FYI, this could have been done with `containsOnly` as well. Which ignores the order of the entries, but no big deal